easy-retweet.php

```diff
@@ -148,12 +148,12 @@ function save_postdata( $post_id ) {
 
 		// verify this came from the our screen and with proper authorization,
 		// because save_post can be triggered at other times
-
-		if ( ! wp_verify_nonce( $_POST['retweet_noncename'], plugin_basename( __FILE__ ) ) ) {
+		
+		if ( isset( $_POST['retweet_noncename'] ) && ! wp_verify_nonce( $_POST['retweet_noncename'], plugin_basename( __FILE__ ) ) ) {
```

Comment:
Using `isset` is correct, but the conditions are wrong.

Basically we should return post id if the nonce verification fails. To check for nonce we need the `$_POST` variable to contain `retweet_noncename`.

Now the new change will not return the post id if the `retweet_noncename` is not set. Instead it will allow the function to run.